signals/signal_tester.py

```diff
@@ -0,0 +1,43 @@
+from signals import signal_exporter
```

Comment:
I really appreciate that you have this script here to show usage and for testing, but can we take it out of the branch? Maybe a make it a GitHub gist? I don't think it belongs in the source code. We can for sure add stuff like this when we get unit testing support.

Reply:
Yeah, I'll manage the script organization and documentation and testing and stuff once I've figured out where this work will actually ending up living. It may end up staying in this repo, or it may end up in its own repo (I'll prolly ask abt it at the tools meeting this week)